peer-review-bundle-spec.md

```diff
@@ -0,0 +1,147 @@
+# Qwiklabs PeerReview Bundle Specification
+
+**Version 1**
+
+> This is a DRAFT document. We welcome feedback as this format evolves.
+
+## `qwiklabs.yaml` Structure
+
+Here's a sample `qwiklabs.yaml` file with all nested details removed to make it
+easier to see the general file structure.
+
+```yaml
+entity_type: PeerReview
+schema_version: 1
+
+default_locale: en
+
+title:
+  locales:
+    en: Create storyboards for your portfolio project
+
+# The primary instruction content for this peer review
+instruction: ...
+
+review_criteria:
+  locales:
+    en: |
+      There are a total of 6 points for this activity. At least two of your peers will evaluate your project. Your final grade will be the median of these scores. You must get 5 out of 6 total points to pass.
+
+prompts: ...
+
+```
+
+Note that all of the main chunks of localized content (title, review_criteria, text, stem and title) are HTML content that may be displayed
+in various contexts. All of these chunks will be sanitized according to the
+restricted set in the [HTML spec](./html/html-spec.md).
+
+### Attribute specification
+
+The full specification is as follows:
+
+attribute               | required | type       | notes
```

Comment:
Just confirming that every item in this table required. I am asking since I don't this is common in other similar specs.

Reply:
i added two optional settings the author could add. (similar from Cousera: https://screenshot.googleplex.com/3DuUuWtN2ZPEAyT) 

let me know if you like to have more info. thank you.